cmd/git-sync/main.go

```diff
@@ -251,6 +259,11 @@ func main() {
 		}
 	}
 
+	if *flSubmoduleMode != SubmoduleModeRecursive && *flSubmoduleMode != SubmoduleModeOff && *flSubmoduleMode != SubmoduleModeShallow {
```

Comment:
Can we make this more readable like:

```
map[string]bool submoduleModes = {
    SubmoduleModeRecursive: true,
    SubmoduleModeShallow: true,
    SubmoduleModeOff: true,
}
 if !submoduleModes[*flSubmoduleMode] {
    error and exit
}
```

Reply:
or 
```
switch *flSubmoduleMode {
case SubmoduleModeRecursive, SubmoduleModeShallow, SubmoduleModeOff:
default:
    error, exit
}
```

Reply:
oh yep switches are fun. when i was doing this a couple months ago i was deep in python and forgot about em.